GliaWidgets/Public/Glia/Glia.Deprecated.swift

```diff
@@ -184,6 +184,30 @@ extension Glia {
             try completion()
         }
     }
+
+    /// Deprecated, use ``Deprecated, use ``Glia.configure(with:uiConfig:assetsBuilder:completion`` and ``Glia.startEngagement(engagementKind:in:theme:features:sceneProvider:)`` instead.`` instead.
```

Comment:
This description is wrong, it differs with the one in the `@available` and also it says Deprecated twice

Reply:
Fixed e5313918ef176901f5676d9fec12cada234bcea6